pkg/clusteragent/README.md

```diff
@@ -0,0 +1,80 @@
+# Datadog Cluster Agent - DCA
+
+[![CircleCI](https://circleci.com/gh/DataDog/datadog-agent/tree/master.svg?style=svg&circle-token=dbcee3f02b9c3fe5f142bfc5ecb735fdec34b643)](https://circleci.com/gh/DataDog/datadog-agent/tree/master)
+[![Build status](https://ci.appveyor.com/api/projects/status/kcwhmlsc0oq3m49p/branch/master?svg=true)](https://ci.appveyor.com/project/Datadog/datadog-agent/branch/master)
+[![GoDoc](https://godoc.org/github.com/DataDog/datadog-agent?status.svg)](https://godoc.org/github.com/DataDog/datadog-agent)
+
+The Datadog Cluster Agent (referred to as DCA) faithfully collects events and metrics and brings them to
+[Datadog](https://app.datadoghq.com) on your behalf so that you can do something
+useful with your monitoring and performance data.
+
+The purpose of the DCA is to be used alongside of an Orchestrator.
+So far, only Kubernetes is supported.
+
+The DCA has two goals:
+ * Be the main interface with the API server to collect and forward events.
+ * Implement the backend for applications requiring a single interface, for instance:
+    -  Keep a map of Container and their metadata associated that would otherwise need to be queried by each agent to the API Server.
+    -  Collect the Control Pane health check
+
+
+The present repository contains the source code of the Datadog Cluster Agent version 6,
+currently in Alpha.
+
+## Getting started
+
+To build the Agent you need:
```

Comment:
You can probably link to the getting started of the main agent here as well, so we can avoid maintaining 2 docs